Source/Public/Build-Bicep.ps1

```diff
@@ -0,0 +1,91 @@
+<#
+.SYNOPSIS
+    Compiles bicep files to JSON ARM Templates
+.DESCRIPTION
+    Build-Bicep is equivalent to bicep build but with the possibility to compile all .bicep files in a directory.
+.PARAMETER Path
+    Specfies the path to the directory or file that should be compiled
+.PARAMETER ExcludeFile
+    Specifies a .bicep file to exclude from compilation
+.PARAMETER GenerateParameterFile
+    The -GenerateParameterFile switch generates a ARM Template paramter file for the compiled template
+.EXAMPLE
+    Build-Bicep vnet.bicep
+    Compile single bicep file in working directory
+.EXAMPLE
+    Build-Bicep 'c:\bicep\modules\vnet.bicep'
+    Compile single bicep file in different directory
+.EXAMPLE
+    Build-Bicep
+    Compile all .bicep files in working directory
+.EXAMPLE
+    Build-Bicep -Path 'c:\bicep\modules\'
+    Compile all .bicep files in different directory
+.EXAMPLE
+    Build-Bicep -ExcludeFile vnet.bicep
+    Compile all .bicep files in the working directory except vnet.bicep
+.EXAMPLE
+    Build-Bicep -Path 'c:\bicep\modules\' -ExcludeFile vnet.bicep
+    Compile all .bicep files in different directory except vnet.bicep
+.EXAMPLE
+    Build-Bicep -GenerateParameterFile
+    Compile all .bicep files in the working directory and generates a parameter files
+.NOTES
+    Go to module repository https://github.com/StefanIvemo/BicepPowerShell for detailed info, reporting issues and to submit contributions.
+#>
+function Build-Bicep {
+    [CmdletBinding(DefaultParameterSetName = 'Default')]
+    [Alias('Invoke-BicepBuild')]
+    param (
+        [Parameter(ParameterSetName = 'Default',Position=1)]
+        [Parameter(ParameterSetName = 'AsString',Position=1)]
+        [string]$Path = $pwd.path,
+
+        [Parameter(ParameterSetName = 'Default',Position=2)]
+        [Parameter(ParameterSetName = 'AsString',Position=2)]
+        [ValidateNotNullOrEmpty()]
+        [string]$OutputDirectory = $pwd.Path,
```

Comment:
The default value for `$OutputDirectory` should not be `$pwd.Path`. The default behavior should be to output the compiled json to the same directory as the provided `.bicep` file.